utils/bench_runner.py

```diff
@@ -143,7 +143,7 @@ def BENCHMARK_COMMAND_ARGS(file_path):
 
 
 def HYPERFINE_COMMAND(file_out):
-    return f'hyperfine --warmup 1 --runs 5 --cleanup "sleep 1" --show-output --export-json {file_out}'
+    return f'hyperfine --ignore-failure --warmup 1 --runs 5 --cleanup "sleep 1" --show-output --export-json {file_out}'
```

Comment:
I am a little worried about this because we did not get to the bottom of why windows is returning a nonzero error code but the other operating systems are not. If you are trying to get this out the door, I would only use this `ignore-failure` for windows and then add a todo here linking to a jira ticket to get to the bottom of it. I was able to recreate the error using a windows 10 VM with VMWare.

Reply:
Did end up putting (continue-on-error) in the swift-toolbox-ci, (should probably remove that as well)
Created jira ticket here https://swift-nav.atlassian.net/browse/CPP-910 for the VM error.

Think we can just remove ignore-failure tbh should be ok (maybe will find out, tryna avoid messing with toolbox CI too much again)